repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -3495,62 +3503,60 @@ public void deleteClassificationOnlyPropagation(String deletedEdgeId) throws Atl
             return;
         }
 
```

Comment:
Here please add one check if the classification still attached to that edge or not, what happens sometime classification might be deleted from the vertex attached to that edge by classification deletion or only propagation deletion of other edge, in that case we will not want to continue traversing. Adding that check could optimize some cases.